packages/nc-gui/composables/useData.ts

```diff
@@ -403,6 +403,8 @@ export function useData(args: {
       if (row.rowMeta) row.rowMeta.saving = false
     }
 
+    // reload data since row update may change the other columns data( formula, QrCode, Barcode, Rollup, Checkbox, User, Auto Updated Datetime, etc...)
+    await callbacks?.loadData?.()
```

Comment:
In the block above we try to update those to avoid refreshing all page with every bulk update, isn't it possible to avoid calling this

Reply:
If there is column which refers to the updated column(eg: Formula, QrCode, Barcode, etc...) it won't reload on clearing the cell. If we are clearing single cell it's updating row using the response data but for bulk update we don't have those information.

Reply:
We can skip it when unnecessary by checking presence of any such column